### PR TITLE
#166880071 setup sass 

### PR DESCRIPTION
**Pull request by NicholusMuwonge**
#### What does this PR do?
- allows using of sass with WebPack 

#### Description of Task to be completed?
- add the necessary loaders to compile the files with extensions .css and .scss to regular CSS.

#### How should this be manually tested?
- Clone this repo and run ``` npm install``` and then ``` npm start```.
- You will see that the styling in the .scss file is picked up by the template and that the word color is purple.

Screenshot 2019-07-03 at 17 17 30

#### What are the relevant pivotal tracker stories?
[#166880071](https://www.pivotaltracker.com/story/show/166880071)

### CODE REVIEW

**Review by richien — APPROVED** [review]

**Review by frankopkusianwar — APPROVED** [review]
LGTM

File: src/components/App.js [hunk 1]
```diff
@@ -15,20 +9,9 @@ class App extends Component {
...
-          <Switch>
-            <Route exact path="/" component={Home} />
-            <Route path="/login" component={Login} />
-            <Route component={notFound} />
-          </Switch>
-        </div>
-      </BrowserRouter>
+      <div>
+        <h1 className="coloring">Welcome to Authors Haven</h1>
+      </div>
```
Comment (MuhanguziDavid) [inline src/components/App.js]: @NicholusMuwonge , what was the reason for removing the routes!?

Comment (NicholusMuwonge) [reply src/components/App.js]: It was a conflict and Initially, I had demonstrated it this way.
But I can work around it with the routes

Comment (NicholusMuwonge) [reply src/components/App.js]: I have reinstated the routes and Implemented it that way.

Comment (MuhanguziDavid) [reply src/components/App.js]: Create a `routes` folder with an `index` file containing the routes to the home page, login, and any other route.
Call the routes index class/functional class in the `App.js` (think of how the app will be set up, not just your task)

Comment (NicholusMuwonge) [reply src/components/App.js]: I have reacted to this feedback kindly please look into it.

**Pull request merged**